### PR TITLE
Measure history ldc2

### PR DESCRIPTION
**Pull request by edeyoung**

### CODE REVIEW

File: app/assets/javascripts/views/patient_builder/patient_version_compare.js.coffee [hunk 1]
```diff
@@ -8,69 +8,64 @@ class Thorax.Views.PatientBuilderCompare extends Thorax.Views.BonnieView
...
+    populationLogicView
+  
+  # creates a patient logic view based off of the current patient state.
+  _getLiveView: (measure) =>
+    populationLogicView = new Thorax.Views.BuilderPopulationLogic(isCompareView: true, suppressDataCriteriaHighlight: true)
+    populationLogicView.setPopulation measure.get('displayedPopulation')
+    populationLogicView.showRationale @model
+    populationLogicView
   
 # Modified Thorax.Views.BuilderPopulationLogic that accepts new Thorax.Models.cachedResult
```
Comment (hossenlopp) [inline app/assets/javascripts/views/patient_builder/patient_version_compare.js.coffee]: Typo `cachedResult` => `CachedResult`

File: app/assets/javascripts/views/patient_builder/patient_version_compare.js.coffee [hunk 2]
```diff
@@ -8,69 +8,64 @@ class Thorax.Views.PatientBuilderCompare extends Thorax.Views.BonnieView
...
     else
-      @populationPresentAfterUpload = false
+      @populationLogicViewAfter = @_getLiveView(@measure)
 
-    @render()  
+    @render()
+  
+  # creates a patient logic view based off of a patient snapshot rather than the current
+  # patient state.
+  _getSnapshotView: (population, summaryCachedResult) =>
```
Comment (hossenlopp) [inline app/assets/javascripts/views/patient_builder/patient_version_compare.js.coffee:48]: Is prepending a class method with an underscore a way to mark methods as 'private' or should only be used from inside the class?

**Review by hossenlopp — CHANGES_REQUESTED** [review]
This looks good to me. I just had one question about coffeescript class conventions. One small typo in comment. I tested this. It looks like it doesn't break the patient builder highlighting and fixes the latest compare coming from measure history using the current state of the patient.

I did notice the highlighting in Measure Observations didn't work on CMS111 but I presume thats because the crazy stuff from it calculating median times there. I haven't tried it on master/production.

File: app/assets/javascripts/views/patient_builder/patient_version_compare.js.coffee (hunk 2 above)
Comment (edeyoung) [reply app/assets/javascripts/views/patient_builder/patient_version_compare.js.coffee:48]: yes - i looked into how to do things as private in coffeescript. signalling seemed like the easiest way - or doing nothing at all. curious what you think we should do here or if we should just do nothing.

Comment (hossenlopp) [reply app/assets/javascripts/views/patient_builder/patient_version_compare.js.coffee:48]: I'm cool with `_` meaning only for use in this class. It's a convention I have used elsewhere.

**Review by hossenlopp — APPROVED** [review]

**Pull request merged**